docs/ru/I18n.md

```diff
@@ -0,0 +1,104 @@
+## Internationalization
+
+Day.js has great support for internationalization.
+
+But none of them will be included in your build unless you use that.
+
+By default, Day.js comes with English (United States) locale.
+
+You can load multiple locales and switch between them easily.
+
+[List of supported locales](../../src/locale)
+
+You are super welcome to add a locale by opening a pull request :+1:
+
+## API
+
+#### Changing locale globally
+
+* Returns locale string
+
+```js
+import 'dayjs/locale/es'
+import de from 'dayjs/locale/de'
+dayjs.locale('es') // use loaded locale globally
+dayjs.locale('de-german', de) // use locale and update default name string
+const customizedLocaleObject = { ... } // More details can be found in Customize section below.
+dayjs.locale(customizedLocaleObject) // use customize locale
+```
+
+* Changing the global locale doesn't affect existing instances.
+
+#### Changing locales locally
+
+* Returns CURRENT `Dayjs` object.
```

Comment:
Just a reminder here, in current version, changed to `Returns a new Dayjs object by switching to new locale`.

Reply:
Thank you, @iamkun.
But I don't promise to complete it in the next two-three days. (Working and studying at uni take all my free time). Probably it will be ready at next Sunday